### PR TITLE
chore: added document content stream

### PR DESCRIPTION
**Pull request by igorsechyn**

### CODE REVIEW

File: resources/schemas/documentContent.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    },
+    "archivedAt": {
+      "type": ["null", "string"]
+    },
+    "content": {
+      "type": ["null", "string"]
+    },
+    "contentData": {
+      "type": ["null", "object"]
+    },
```
Comment (urossmolnik) [inline resources/schemas/documentContent.json:24]: This field is deprecated and will soon be removed. I don't think we should introduce it to Airbyte connector?

File: src/client/LinearClient.ts [hunk 2]
```diff
@@ -174,12 +175,19 @@ export class LinearClient {
...
   public async documents(): Promise<Document[]> {
     return await this.fetchEntities<Document>("document");
   }
 
+  /**
+   * @returns List of all document content entities in organization.
+   */
+  public async documentContents(): Promise<Document[]> {
+    return await this.fetchEntities<Document>("documentcontent");
+  }
```
Comment (urossmolnik) [inline src/client/LinearClient.ts:189]:
```suggestion
  public async documentContents(): Promise<DocumentContent[]> {
    return await this.fetchEntities<DocumentContent>("documentcontent");
  }
```

File: src/client/types.ts [hunk 3]
```diff
@@ -371,13 +370,24 @@ export interface Document {
...
   updatedById?: string;
   projectId?: string;
   slugId?: string;
 }
+
+export interface DocumentContent {
+  id?: string;
+  organizationId: string;
+  content?: string;
+  contentData?: string;
```
Comment (urossmolnik) [inline src/client/types.ts:385]: Also remove from type

File: resources/schemas/project.json [hunk 4]
```diff
@@ -16,9 +16,6 @@
     "name": {
       "type": "string"
     },
-    "description": {
-      "type": "string"
-    },
```
Comment (urossmolnik) [inline resources/schemas/project.json:21]: @igorsechyn I don't you should be removing this.

**Review by urossmolnik — APPROVED** [review]

File: resources/schemas/project.json (hunk 4 above)
Comment (igorsechyn) [reply resources/schemas/project.json:21]: I thought project description is becoming a document content too. Document content has a project if reference

**Pull request merged**